infra/lib/api-stack.ts

```diff
@@ -700,6 +709,31 @@ export class APIStack extends Stack {
     return cdaToVisualizationLambda;
   }
 
+  private setupCWDocContribution(
+    lambdaLayers: lambda.ILayerVersion[],
+    alarmAction: SnsAction | undefined,
+    api: apig.RestApi,
+    authorizer: apig.IAuthorizer,
+    oauthScopes: cognito.OAuthScope[]
+  ): apig.Resource {
+    const cwLambda = createLambda({
+      stack: this,
+      name: "CommonWellDocContribution",
+      runtime: lambda.Runtime.NODEJS_16_X,
+      entry: "cw-doc-contribution",
+      layers: lambdaLayers,
+      alarmSnsAction: alarmAction,
+    });
+
+    const cwResource = api.root.addResource("cw");
```

Comment:
It'd be good to add this to a grouping path, something like `/documents`, `/contribution` or something that represents what this endpoint does. Just so later we can add other HIEs to the same path.